tests/unit/test_dockerclient.py

```diff
@@ -111,3 +111,82 @@ def list_in(a, b):
     return len(a) <= len(b) and any(
         map(lambda x: b[x : x + len(a)] == a, range(len(b) - len(a) + 1))
     )
+
+
+class TestCommandLine(unittest.TestCase):
```

Comment:
Nit: While fixing the Linter errors, could you rename this test suite into something more suiting? Like TestPortMappings or something like that. I know it has been named like this in the old location, but the name does not make sense here.

Reply:
Done.